packages/xs/xapi-idl.1.90.1/opam

```diff
@@ -10,10 +19,11 @@ remove: [
 depends: [ "uri"
            "re"
            "cmdliner"
-           "cohttp" {>= "0.15.2"}
+           "cohttp"
            "xmlm"
-           "rpc" {>= "1.9.4"}
+           "rpc" {>= "1.5.3"}
```

Comment:
No, but there are many other `opam` files in `xs-opam` with version constraints as we usually copy the original opam files into the repo.

I think it is a useful safety check to have these version constraints, because this can warn us in case we include the wrong version of a package that is incompatible with another one in the repo. For example, OPAM would complain if we included rpc 1.0.0 in our repo, instead of the newest version. So it might be useful to preserve these version constraints as a sanity check, as they tell us something about which packages our current one is or isn't compatible with.